docs/how-to/quantization.md

```diff
@@ -41,76 +41,26 @@ There are 3 ways of quantizing a model: dynamic, static and quantize-aware train
 
 * **Quantize-Aware training quantization**: The quantization parameter of activation are calculated while training, and the training process can control activation to a certain range.
 
-### List of Supported Quantized Ops
-The following ops were chosen as phase 1 ops because these ops consume the most compute and power in most CNN models.
- * Conv
- * MatMul
- * MaxPool
- * Relu
- * Clip
- * Add (Experimental)
- * Mul (Experimental)
-
- ### Quantization and model opset versions
- Quantization ops were introduced in ONNX opset version 10, so the model which is being quantized must be opset 10 or higher. If the model opset version is < 10 then the model should be reconverted to ONNX from its original framework using a later opset.
-
-### Quantization and Graph Optimization
-Quantization and graph optimizations may not always work together. The model owner should be aware of this and run perf evaluations to understand which technique provides the best performance for their model.
-
-* **Quantizing an optimized model**
+### Method selection
+The main difference between dynamic quantization and static quantization is how scale and zero point of activation is calculated. For static quantization, they are calculated offline with calibration data set. All the activations have same scale and zero point. While for dynamic quantization, they are calculated on flight and will be specific for each activation, thus they are more accurate but introduce extra computation overhead.
 
-    If a model is optimized using level 99 (i.e. all possible optimizations are run on that model) then it is possible that quantization cannot be applied on this model anymore. In this case, running the quantization script will not affect the model.
+In general, it is recommended to use dynamic quantization for RNN and transformer-based models, and static quantizaiton for CNN models.
 
-* **Optimizing a quantized model**
+If both post-training quantization can not meet your accuracy goal, you can try quantization-aware training to retrain the model. OnnxRuntime doesn't provide retrain capability now. You can retrain you model with original framework and then converted it back to ONNX. 
 
-    The same holds the other way around. After quantizing a model, some graph optimizations which otherwise might have been applicable may not be applicable anymore.
+## List of Supported Quantized Ops
+Please refer to [registry](https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/python/tools/quantization/registry.py) for the list of supported Ops.
 
+## Quantization and model opset versions
+ Quantization ops were introduced in ONNX opset version 10, so the model which is being quantized must be opset 10 or higher. If the model opset version is < 10 then the model should be reconverted to ONNX from its original framework using a later opset.

```

Comment:
We can keep it short and just say that opset 10 is required. 
"Models must be opset10 or higher to be quantized. Models with opset < 10 must be reconverted to ONNX from its original framework using a later opset."